src/percy-node-client.js

```diff
@@ -305,12 +305,12 @@ async function checkBuildStatus(buildId, numRetries, resolve) {
   if (state == 'processing' || state == 'pending') {
       retry(buildId, numRetries, resolve);
   } else if (state == 'finished'){
-    const totalDiffs = attributes['total-comparisons-diff'];
-      if (totalDiffs) {
+    const totalUnreviewed = attributes['total-snapshots-unreviewed'];
```

Comment:
Please add a quick code comment here explaining that unreviewed diffs are diffs which haven't been approved in the percy UI.